shotover-proxy/tests/cassandra_int_tests/basic_driver_tests.rs

```diff
@@ -0,0 +1,123 @@
+use crate::cassandra_int_tests::CassandraTestContext;
+use crate::helpers::ShotoverManager;
+use test_helpers::docker_compose::DockerCompose;
+
+use anyhow::Result;
+use cassandra_cpp::*;
+use std::thread;
+use tracing::{debug, info, warn};
+
+fn test_create_keyspace() {
+    let mut query = stmt!(
+        "CREATE KEYSPACE IF NOT EXISTS cycling WITH REPLICATION = { 'class' : 'SimpleStrategy', 'replication_factor' : 1 };"
+    );
+    let ctx = CassandraTestContext::new();
+    let mut result = ctx.session.execute(&query).wait().unwrap();
+    debug!("{:?} result {:?}", thread::current().id(), result);
+    assert_eq!(result.row_count(), 0);
+
+    query = stmt!("SELECT release_version FROM system.local");
+    result = ctx.session.execute(&query).wait().unwrap();
+    debug!("{:?} result {:?}", thread::current().id(), result);
+    assert_eq!(result.row_count(), 1);
+    assert_eq!(
+        result
+            .first_row()
+            .unwrap()
+            .get_column(0)
+            .unwrap()
+            .get_str()
+            .ok(),
+        Some("3.11.10")
+    );
+
+    query = stmt!("SELECT keyspace_name FROM system_schema.keyspaces;");
+    result = ctx.session.execute(&query).wait().unwrap();
+    debug!("{:?} result {:?}", thread::current().id(), result);
+    assert_eq!(result.row_count(), 6);
+    assert_eq!(
+        result
+            .first_row()
+            .unwrap()
+            .get_column(0)
+            .unwrap()
+            .get_str()
+            .ok(),
+        Some("cycling")
+    );
+}
+
+#[test]
+fn test_basic_connection() -> Result<()> {
```

Comment:
No need to return `Result<()>` here. Failures are indicated in tests by panicking.